### PR TITLE
Responsive Dashboard

### PR DESCRIPTION
**Pull request by amitojsingh**
1. Responsive Dashboard
2. Responsive Word Pages
3. Responsive Question Pages

### CODE REVIEW

**Comment by MySmilingTurban** [comment]
The words you learnt box collapses when width goes below 768

https://github.com/KhalisFoundation/gurmukhi-web/assets/51584037/87a95a49-dc8e-417f-af8e-b4b04b0f68da

**Comment by MySmilingTurban** [comment]
Another thing, below 341 width, it squeezes the profile icon in the header
Also, make sure the background remains as parallax because right now I can see it repeat

https://github.com/KhalisFoundation/gurmukhi-web/assets/51584037/bf301324-a976-450a-bf2c-6acd600197e3

**Comment by amitojsingh** [comment]
Fixed @MySmilingTurban  Veer ji can you try now.

**Comment by amitojsingh** [comment]
@MySmilingTurban  Veer ji I changed the design because the text was getting overlapped with the header on scroll which is poor typography so to avoid that I have added a background to header and footer which is very much similar with minor difference.

**Comment by MySmilingTurban** [comment]
Please don't add background image to header and footer, it messes up the design

https://github.com/KhalisFoundation/gurmukhi-web/assets/51584037/a59a8774-1b9d-40b9-bb8e-23e4c41d870f

**Comment by MySmilingTurban** [comment]
Oh, can you can there background to blur instead of a solid picture, cuz it feels odd

**Comment by MySmilingTurban** [comment]
you can remove background picture from footer, as it will not overlap any text as it is fixed to bottom of the page

**Comment by amitojsingh** [comment]
fixed veer ji

File: src/components/questions/multiple-choice.tsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default function MultipleChoiceQuestion({
   return (
     <div className='flex flex-col items-left justify-evenly text-center'>
       <div className='flex flex-row items-center justify-between gap-5 rounded-lg p-4'>
-        <h1 className={'text-5xl gurmukhi text-black font-semibold'}>
+        <h1 className={'xl:text-5xl text-4xl gurmukhi text-black font-semibold leading-snug'}>
```
Comment (MySmilingTurban) [inline src/components/questions/multiple-choice.tsx:110]: nit: media query class name should be written after general ones

File: src/assets/icons/Shabadavali.tsx [hunk 2]
```diff
@@ -4,8 +4,8 @@ import { useTranslation } from 'react-i18next';
 export default function Shabadavali() {
   const { t: text } = useTranslation();
   return (
-    <div className="flex items-center justify-center w-12 h-12 mr-4 rounded-full logo-bg">
-      <span className="text-2xl text-white gurmukhi mustard">{text('SHASHA')}</span>
+    <div className='flex items-center justify-center md:w-12 md:h-12 w-5 h-5 mr-4 rounded-full logo-bg'>
```
Comment (MySmilingTurban) [inline src/assets/icons/Shabadavali.tsx:7]: media query class name should be written after general ones

File: src/pages/dashboard/components/coinbox/index.tsx [hunk 3]
```diff
@@ -11,10 +11,10 @@ function CoinBox({ commonStyle }: { commonStyle: string }) {
   return (
     <div className={commonStyle}>
       <div className='flex justify-center items-center h-full'>
-        <div className='my-4'>
+        <div className=''>
```
Comment (MySmilingTurban) [inline src/pages/dashboard/components/coinbox/index.tsx:14]: nit: remove empty classname

File: src/pages/wincoin/index.tsx [hunk 4]
```diff
@@ -59,7 +59,7 @@ function WinCoin() {
     <div className='nanakback h-full bg-cover w-full'>
       <Meta title={title} description={description} />
       <div className='flex flex-col text-center justify-evenly h-4/5 recoleta'>
-        <div className=' mx-auto bg-cyan-50 w-1/4 rounded-xl'>
+        <div className=' mx-auto bg-cyan-50 xl:w-1/4 w-2/4 rounded-xl'>
```
Comment (MySmilingTurban) [inline src/pages/wincoin/index.tsx:62]: nit: media query class name should be written after general ones

File: src/pages/word/information/index.tsx [hunk 5]
```diff
@@ -78,16 +78,16 @@ export default function Information() {
   return (
     <div className='flex flex-col items-center w-full h-full justify-between gap-5'>
       <Meta title={title} description={description} />
-      <div className='flex flex-col h-3/4 justify-between items-center'>
+      <div className='flex flex-col h-3/4 justify-between items-center '>
```
Comment (MySmilingTurban) [inline src/pages/word/information/index.tsx:81]: nit: empty space

File: src/pages/word/information/index.tsx [hunk 6]
```diff
@@ -78,16 +78,16 @@ export default function Information() {
...
           className='w-3/5 h-6'
           src='/icons/pointy_border.svg'
           alt='border-top'
           width={200}
           height={200}
         />
-        <div className='flex flex-row items-center justify-between h-full gap-10 w-full'>
-          <div className='flex flex-col items-left justify-evenly w-1/2 h-full'>
+        <div className='flex flex-col xl:flex-row items-center justify-between h-full gap-10 w-full py-10 overflow-y-auto'>
+          <div className='flex flex-col items-left justify-evenly xl:w-1/2 w-3/4 h-full'>
```
Comment (MySmilingTurban) [inline src/pages/word/information/index.tsx:90]: nit: media query class name should be written after general ones

File: src/pages/word/information/index.tsx [hunk 7]
```diff
@@ -146,7 +148,7 @@ export default function Information() {
             </div>
             <div className='flex items-center justify-around gap-5 w-full h-1/2'>
               <div
-                className={`w-2/5 h-full cardImage bg-cover bg-sky-100 bg-blend-soft-light hover:bg-sky-50 border-2 border-sky-200 shadow-lg rounded-lg ${
+                className={`lg:w-2/5 w-5/6 h-full cardImage bg-cover bg-sky-100 bg-blend-soft-light hover:bg-sky-50 border-2 border-sky-200 shadow-lg rounded-lg ${
```
Comment (MySmilingTurban) [inline src/pages/word/information/index.tsx:151]: nit: media query class name should be written after general ones

File: src/pages/word/information/index.tsx [hunk 8]
```diff
@@ -172,7 +174,7 @@ export default function Information() {
                 </div>
               </div>
               <div
-                className={`w-2/5 h-full cardImage bg-cover bg-sky-100 bg-blend-soft-light hover:bg-sky-50 border-2 border-sky-200 shadow-lg rounded-lg ${
+                className={`lg:w-2/5 w-5/6 h-full cardImage bg-cover bg-sky-100 bg-blend-soft-light hover:bg-sky-50 border-2 border-sky-200 shadow-lg rounded-lg ${
```
Comment (MySmilingTurban) [inline src/pages/word/information/index.tsx:177]: nit: media query class name should be written after general ones

File: src/styles/globals.scss [hunk 9]
```diff
@@ -30,18 +32,23 @@ html,body,#root{
 * {
   font-family: 'Recoleta';
 }
+.header-bg{
+  background:rgb(244 250 255 / 81%);
+}
+.back-bg {
+  background: rgb(184 214 234);
+}
+.back-bg-image {
```
Comment (MySmilingTurban) [inline src/styles/globals.scss:41]: nit: add empty line after each class style

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]

File: src/components/questions/multiple-choice.tsx (hunk 1 above)
Comment (amitojsingh) [reply src/components/questions/multiple-choice.tsx]: fixed

File: src/assets/icons/Shabadavali.tsx (hunk 2 above)
Comment (amitojsingh) [reply src/assets/icons/Shabadavali.tsx]: fixed

File: src/pages/dashboard/components/coinbox/index.tsx (hunk 3 above)
Comment (amitojsingh) [reply src/pages/dashboard/components/coinbox/index.tsx]: fixed

File: src/pages/wincoin/index.tsx (hunk 4 above)
Comment (amitojsingh) [reply src/pages/wincoin/index.tsx]: fixed

File: src/pages/word/information/index.tsx (hunk 5 above)
Comment (amitojsingh) [reply src/pages/word/information/index.tsx]: fixed

File: src/pages/word/information/index.tsx (hunk 6 above)
Comment (amitojsingh) [reply src/pages/word/information/index.tsx]: fixed

File: src/pages/word/information/index.tsx (hunk 7 above)
Comment (amitojsingh) [reply src/pages/word/information/index.tsx]: fixed

File: src/pages/word/information/index.tsx (hunk 8 above)
Comment (amitojsingh) [reply src/pages/word/information/index.tsx]: fixed

File: src/styles/globals.scss (hunk 9 above)
Comment (amitojsingh) [reply src/styles/globals.scss]: fixed

File: src/assets/icons/Shabadavali.tsx [hunk 10]
```diff
@@ -4,8 +4,8 @@ import { useTranslation } from 'react-i18next';
 export default function Shabadavali() {
   const { t: text } = useTranslation();
   return (
-    <div className="flex items-center justify-center w-12 h-12 mr-4 rounded-full logo-bg">
-      <span className="text-2xl text-white gurmukhi mustard">{text('SHASHA')}</span>
+    <div className='flex items-center justify-center w-5 h-5 mr-4 rounded-full logo-bg md:w-12 md:h-12 '>
```
Comment (MySmilingTurban) [inline src/assets/icons/Shabadavali.tsx:7]: nit: remove empty space in end of classname

File: src/components/questions/multiple-choice.tsx [hunk 11]
```diff
@@ -107,7 +107,7 @@ export default function MultipleChoiceQuestion({
   return (
     <div className='flex flex-col items-left justify-evenly text-center'>
       <div className='flex flex-row items-center justify-between gap-5 rounded-lg p-4'>
-        <h1 className={'text-5xl gurmukhi text-black font-semibold'}>
+        <h1 className={' gurmukhi text-black font-semibold leading-snug xl:text-5xl text-4xl'}>
```
Comment (MySmilingTurban) [inline src/components/questions/multiple-choice.tsx:110]: change the classname to 
`gurmukhi text-black font-semibold leading-snug text-4xl xl:text-5xl`
(remove empty space and order of media query)

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]

File: src/assets/icons/Shabadavali.tsx (hunk 10 above)
Comment (amitojsingh) [reply src/assets/icons/Shabadavali.tsx]: fixed

File: src/components/questions/multiple-choice.tsx (hunk 11 above)
Comment (amitojsingh) [reply src/components/questions/multiple-choice.tsx]: fixed

**Comment by MySmilingTurban** [comment]
Please fix the image pages for word

Screenshot 2024-03-02 at 5 51 55 pm

**Comment by amitojsingh** [comment]
@MySmilingTurban veer ji looks like you have zoomed in on the image can you verify with 100% zoom?

**Comment by amitojsingh** [comment]
@MySmilingTurban Veer ji please review

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
Looks good to me!
One minor issue with the highlighted element in the attached screenshot, can we make it responsive too

Screenshot 2024-03-10 at 10 20 23 am

**Comment by amitojsingh** [comment]
fixed veer ji @MySmilingTurban

**Review by MySmilingTurban — APPROVED** [review]
Looks good to me!

**Pull request merged**